src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Breadcrumb/BitBreadcrumbDemo.razor.cs

```diff
@@ -8,14 +8,16 @@ namespace Bit.Client.Web.BlazorUI.Playground.Web.Pages.Components.Breadcrumb
 {
     public partial class BitBreadcrumbDemo
     {
+        public string onClickValue { get; set; } = string.Empty;
```

Comment:
`OnClickValue`
Plz, use pascal case for property name

Reply:
Notice that this property has the `public` access modifier. All of the public fields must be written in a pascal case format.